README.md

```diff
@@ -8,154 +8,23 @@ A WooCommerce plugin that automatically generates and delivers product feeds to
 
 ### Core Features
 
-- **Complete OpenAI Specification Coverage** - All required and optional fields supported
-- **Real-time Updates** - Products sync within 30 seconds of changes
-- **Scheduled Delivery** - Automatic full feed updates every 15 minutes
-- **Per-Product Control** - Override global settings for individual products
-- **Feed Validation** - Built-in validation against OpenAI specifications
-- **Multiple Formats** - JSON, CSV, XML, TSV export options
-- **Secure API Access** - Admin-only REST endpoints for integration
+- **ProductFeed generation abstraction** to support the large store.
+- **Complete OpenAI Specification Coverage** includes:
+  - All required and optional fields supported
+  - Automatic full feed updates every 15 minutes
+  - Override global settings for individual products
+  - Built-in validation against OpenAI specifications
+- **Pos Catalog generation** 
 
 ## Quick Start
 
 ### Installation
 
-1. **Download** the latest release from GitHub Releases
+1. **Build and get zip file** from the latest `trunk` branch by running: `npm run build`
 2. **Upload** the zip file via WordPress admin (Plugins > Add New > Upload Plugin) or extract to `/wp-content/plugins/`
 3. **Activate** the plugin through WordPress admin
-4. Go to **WooCommerce > Settings > Integrations > ChatGPT**
+4. Go to **WooCommerce > Settings > Integrations > OpenAI**
```

Comment:
Does it require some setup for this settings page/section to show up? I installed the plugin zip on a prod test site and I don't see this section in `WooCommerce > Settings > Integration`:

<img width="1880" height="794" alt="Image" src="https://github.com/user-attachments/assets/e6eb78b8-b798-44ad-8eb7-fadf76b9a2b1" />